en/appendices/4-0-migration-guide.rst

```diff
@@ -179,6 +179,14 @@ Database
 * If your database's timezone does not match PHP timezone then you can use the
   ``DateTime::setTimezone()`` method. See :ref:`datetime-type` for details.
 
+ORM
+---
+
+* ``Table::saveManyOrFail()`` method has been added that will throw ``PersistenceFailedException``
+  with the specific entity that failed in case of an error. The entities are saved transaction safe.
+* ``Table::deleteMany()`` and ``Table::deleteManyOrFail()`` methods have been added for removing many
+  entities at once including callbacks. The entities are removed transaction safe.
```

Comment:
Should these methods also be shown in the orm usage docs alongside the other save and delete methods?